ncbi_genome_download/__main__.py

```diff
@@ -9,21 +9,21 @@
 def main():
     """Build and parse command line"""
     parser = argparse.ArgumentParser()
-    parser.add_argument('domain',
-                        choices=dflt.DOMAIN.choices,
-                        default=dflt.DOMAIN.default,
-                        help='The NCBI "domain" to download (default: %(default)s)')
+    parser.add_argument('group',
```

Comment:
I found in that document ftp://ftp.ncbi.nlm.nih.gov/pub/factsheets/HowTo_Downloading_Genomic_Data.pdf (in the *Genomes FTP Directory Structure* section) that they were refering to it as taxonomic group, thus my naming change.
But I can revert to domain if you'd like to.